ledger/src/blockstore_db.rs

```diff
@@ -148,7 +148,12 @@ pub mod columns {
     pub struct Orphans;
 
     #[derive(Debug)]
-    /// The dead slots column
+    /// The dead slots column.
+    /// This column family tracks whether a slot is dead.
+    ///
+    /// A slot is marked as dead if the validator thinks it will never receive
+    /// all shreds of this slot.  As a result, it is impossible to have a full
+    /// dead slot, and a full slot is never dead.
```

Comment:
Hmm this isn't necesarily true. A full slot can be dead, because there can be runtime errors that cause ReplayStage to mark the slot as dead.

Reply:
Hey!  Thanks for clarifying things.  Will update the comment and include the case you mentioned!